recipes/scxmatch/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "scxmatch" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/bionetslab/scxmatch/releases/download/scxmatch-0.0.1/scxmatch-0.0.1.tar.gz
+  sha256: 19a31ac2c2c5b8a3aafdde5f44bce19e6143f99a612d1c6257fa896db524298f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+requirements:
+  build:
+    - python >=3.9,<3.10
```

Comment:
Is this pin really needed? A noarch-python package is usually not bound to one Python version.